letsencrypt-auto-source/Dockerfile.wheezy

```diff
@@ -6,13 +6,15 @@ FROM debian:wheezy
 # Add an unprivileged user:
 RUN useradd --create-home --home-dir /home/lea --shell /bin/bash --groups sudo --uid 1000 lea
 
-# Install pip, sudo, openssl, and nose:
+# Install pip, sudo, openssl, and pytest:
```

Comment:
The lines immediately below this don't install pytest, so just remove pytest from this line. Same edit for anything starting with "Install pip, " in this change.

Reply:
I made the `pytest` changes.

I think the "Install pip" comments are accurate as we do install `pip` from the package manager, but then update it through `pipstrap` so I changed comments to clarify this.